src/app/openmrs-api/order-resource.service.ts

```diff
@@ -39,20 +42,55 @@ export class OrderResourceService {
   }
 
   public getOrdersByPatientUuid(patientUuid: string, cached: boolean = false, v: string = null):
-  Observable<any> {
+    Observable<any> {
+
+      this.v = 'full';
 
     const url = this.getUrl();
     const params: HttpParams = new HttpParams()
-    .set('patient', patientUuid)
-    .set('v', (v && v.length > 0) ? v : this.v);
+      .set('patient', patientUuid)
+      .set('v', (v && v.length > 0) ? v : this.v);
+    return this.http.get(url, {
+      params: params
+    });
+  }
+
+  public getAllOrdersByPatientUuuid(patientUuid: string, careSettingUuid: string,
+    cached: boolean = false, v: string = null): Observable<any> {
+
+    const url = this.getUrl();
+
+
+    const params: HttpParams = new HttpParams()
+      .set('patient', patientUuid)
+      .set('careSetting', careSettingUuid)
+      .set('status', 'any')
+      .set('v', (v && v.length > 0) ? v : this.v);
+
     return this.http.get(url, {
       params: params
+    }).map((response) => {
+
+      return response;
+    });
+  }
+
+
+
+  public getOrderEntryConfig(): Observable<any> {
+    const url = this.appSettingsService.getOpenmrsRestbaseurl().trim() + 'orderentryconfig';
+    return this.http.get(url).map((response) => {
+      console.log(response, 'initial results');
+      return response;
     });
   }
+
+
   public getOrderByUuid(uuid: string, cached: boolean = false, v: string = null): Observable<any> {
 
     let url = this.getUrl();
     url += '/' + uuid;
+    this.v = 'full';
     // console.log('url', url)
```

Comment:
Remove commented code and also check if you need the full payload or just specific data for this call.